ci/scripts/build.ps1

```diff
@@ -10,9 +10,9 @@ if ($IsWindows) {
 }
 
 if ((qmake --version -split '\n')[1][17] -eq '6') {
-    qmake QMAKE_APPLE_DEVICE_ARCHS="x86_64 arm64" $args[0] PREFIX=$Prefix DEFINES+="$env:NIGHTLYDEFINES"
+    qmake QMAKE_APPLE_DEVICE_ARCHS="x86_64 arm64" $args[0] PREFIX=$Prefix DEFINES+="$env:nightlyDefines"
```

Comment:
The thing about the environment variables getting uppercased was only applicable to `##vso[task.setvariable` hence the case switch here.